tests/test-event.js

```diff
@@ -3,59 +3,22 @@
 // Testing EVENT APIs
 
 var event = require("events");
-
-var total = 0;
-var passed = 0;
-
-function assert(actual, description) {
-    total += 1;
-
-    var label = "\033[1m\033[31mFAIL\033[0m";
-    if (actual === true) {
-        passed += 1;
-        label = "\033[1m\033[32mPASS\033[0m";
-    }
-
-    console.log(label + " - " + description);
-}
-
-function expectThrow(description, func) {
-    var threw = false;
-    try {
-        func();
-    }
-    catch (err) {
-        threw = true;
-    }
-    assert(threw, description);
-}
+var Assert = require("Assert.js");
 
 var eventEmitter = new event();
 
-expectThrow("event: set max of listeners as 'value'", function () {
-    eventEmitter.setMaxListeners("value");
-});
-
-expectThrow("event: set max of listeners as '-1024'", function () {
-    eventEmitter.setMaxListeners(-1024);
-});
-
-expectThrow("event: get max of listeners on event named 'value'", function () {
-    eventEmitter.getMaxListeners("value");
-});
-
 var defaultNum = eventEmitter.getMaxListeners();
-assert(defaultNum === 0, "event: default number of listeners");
+Assert.true(defaultNum === 10, "event: default number of listeners");
```

Comment:
If you use my new patch #650 you wouldn't have to change these lines. (Just do `var assert = require('Assert.js')` above instead.)

So if you're okay with that patch, maybe rebase this on top of it.